third_party/move/move-vm/transactional-tests/tests/tests.rs

```diff
@@ -2,7 +2,175 @@
 // Copyright (c) The Move Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-pub const TEST_DIR: &str = "tests";
-use move_transactional_test_runner::vm_test_harness::run_test;
+use fail::FailScenario;
+use libtest_mimic::{Arguments, Trial};
+use move_bytecode_verifier::VerifierConfig;
+use move_compiler_v2::logging;
```

Comment:
nit: reconsider if we need to add a dependency on compiler for this (I am wondering if this may get us into circular dependency in the future).